### PR TITLE
Makes nanites forced speech / skull echo not spammable + send skull echo message to ghosts

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
did some nanites codes
* Force speech + skull echo is not spammable anymore. they share cooldown now.
* these are still allowed to have more than one program. This is because these are supposed to have multiple answer by given message, but it shouldn't say all things in a short time.
* Cloud number is recorded in investigation
* sends nanites message to ghosts so they can read it too

## Why It's Good For The Game
about the pull request almost explained why we need this.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

you have 4 Nya programs, but only single Nay will be said.

image

This is not spammable too, and ghosts can see message

</details>

## Changelog
:cl:
tweak: Nanites programs 'force speech' and 'skull echo' now share its cooldown with other same programs.
add: ghosts now can read nanites skull echo messages.
/:cl:

### CODE REVIEW

**Comment by CydiaLamiales** [comment]
Wait, does this mean no more ASCII art with nanites?
image

**Comment by EvilDragonfiend** [comment]
> Wait, does this mean no more ASCII art with nanites? image

yeah it doesn't make sense :trollface:

**Review by itsmeow — APPROVED** [review]

File: code/modules/research/nanites/nanite_programs/suppression.dm [hunk 1]
```diff
@@ -162,20 +166,36 @@
...
 	var/sent_message = comm_message
 	if(!comm_message)
 		var/datum/nanite_extra_setting/message_setting = extra_settings[NES_MESSAGE]
 		sent_message = message_setting.get_value()
-	if(host_mob.stat == DEAD)
-		return
 	to_chat(host_mob, "<i>You hear a strange, robotic voice in your head...</i> \"<span class='robot'>[html_encode(sent_message)]</span>\"")
 
+	// send message to ghosts
+	if(!check_if_time_passed(last_time_notified, 2 SECONDS))
```
Comment (itsmeow) [inline code/modules/research/nanites/nanite_programs/suppression.dm]: Use cooldowns for this, they're the same thing but wrappered

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
changed

**Review by itsmeow — APPROVED** [review]

**Pull request merged**